polynote-spark/src/main/scala/polynote/kernel/SparkPolyKernel.scala

```diff
@@ -94,6 +94,9 @@ class SparkPolyKernel(
   private lazy val session = {
     val conf = new SparkConf(loadDefaults = true)
     conf.setIfMissing("spark.master", "local[*]")
+    if (conf.get("spark.master") == "local[*]") {
```

Comment:
This is because sometimes when you run locally it uses the wrong IP address and nothing works.